raiden-cli/src/index.ts

```diff
@@ -15,6 +15,11 @@ import DISCLAIMER from './disclaimer.json';
 import type { Cli } from './types';
 import { setupLoglevel } from './utils/logging';
 
+if (!('RTCPeerConnection' in globalThis)) {
```

Comment:
Do we actually need this condition? This should never exist in a Node environment, shouldn't it?

Reply:
It's improbable, but possible to use this script in a browser runtime, or imported from another script, so I wanted to avoid the double polyfill in case that happens, just in case, but normally, Node will never have WebRTC ootb